keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -832,13 +953,31 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
+Rolling out this feature involves enabling the ReadWriteOncePod feature gate
+across kube-apiserver, kube-scheduler, kubelet, and updating CSI driver and
+sidecar versions. The order in which these are performed does not matter.
+
+The only way this rollout can fail is if a user does not update all components,
+in which case the feature will not work. See the above section on version skews
```

Comment:
The skew deals with the version differences, but gates can be rolled out independently. What happens if the kube-apiserver and kubelet are vN and have the gate enabled, but the scheduler does not?